docs/guides/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_without_the_cli.rst

```diff
@@ -0,0 +1,55 @@
+.. _how_to_guides__creating_and_editing_expectations__how_to_create_a_new_expectation_suite_without_the_cli:
+
+How to create a new Expectation Suite without the CLI
+***************************************************
+
+In some environments, you might not be able to use the :ref:`CLI <command_line>` to create a new Expectation Suite. This guide shows how to create a new Expectation Suite and start adding Expectations using Python code, for example in a Jupyter notebook.
```

Comment:
```suggestion
In some environments, you might not be able to use the :ref:`CLI <command_line>` to create a new Expectation Suite. This guide shows how to create a new Expectation Suite and start adding Expectations using Python code, for example in a Jupyter notebook.

Note: if you want to get started creating Expectations with the fewest possible dependencies, you should check out [How to quickly explore Expectations in a notebook](INSERT REAL LINK HERE). This article assumes that you've already configured a [DataContext](LINK LINK) and (optionally) a [DataSource](LINK LINK).
```

Reply:
I get what you want to do here, but I don't think this is in the spirit of keeping the how-to guides very succinct and focused on one goal. How do you feel about adding this to an "additional material" section at the end instead?

Reply:
Added it to the top as discussed live.